test/proxy.js

```diff
@@ -12,18 +12,18 @@ const process = require('process')
 
 const talkback = require('talkback')
 
-const host = 'https://api.openverse.engineering/v1'
+const host = 'https://api.openverse.engineering'
 
 const tapeNameGenerator = (tapeNumber) => `response-${tapeNumber}`
 
 const updatingTapes = process.argv.includes('--update-tapes')
 const recordMode = updatingTapes
   ? talkback.Options.RecordMode.NEW
-  : talkback.Options.RecordMode.DISABLED
+  : talkback.Options.RecordMode.NEW
 console.log('Record mode: ', recordMode)
 const opts = {
   host,
-  port: 3000,
+  port: 3322,
```

Comment:
:sweat_smile: This seems like a better choice considering how many other things live on port 3000 by default anyway (I know a few that are missing from this list too, like Flask I think?)

https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

In fact, I kind of wonder if we shouldn't switch all of our stuff to use ports in the completely unregistered "temporary services" range of 49152–65535.

Reply:
49152 it is, then :)